bundle/manifests/external-dns-operator_clusterserviceversion.yaml

```
@@ -44,7 +44,11 @@ spec:
       kind: ExternalDNS
       name: externaldnses.externaldns.olm.openshift.io
       version: v1alpha1
-  description: ExternalDNS Operator deploys and manages ExternalDNS, which dynamically manages external DNS records in specific DNS Providers for specific Kubernetes resources.
+  description: "The ExternalDNS Operator deploys and manages ExternalDNS, which dynamically manages DNS records in external DNS Providers for specific Kubernetes resources.\n\n## Prerequisites and Requirements\n
+    The ExternalDNS Operator has minimal cluster level permissions. Unfortunately OLM doesn't allow the local role creation in the operand namespace.\n
+    Therefore make sure the following commands are run against your cluster before the installation of the operator:\n
+    * Create the operand namespace: `oc create ns external-dns`\n
+    * Grant operator the access to operand: `oc apply -f config/rbac/extra-roles.yaml`"
```

Is the user reading this notice in console going to know where to get the  `config/rbac/extra-roles.yaml` file?  Should we provide a full URL?  
```suggestion
    * Grant the operator access to manage operand resources: `oc apply -f https://raw.githubusercontent.com/openshift/external-dns-operator/main/config/rbac/extra-roles.yaml`"
```